docs/workflows.rst

```diff
@@ -320,15 +324,29 @@ Confounds estimation
 
     from fmriprep.workflows.confounds import init_discover_wf
     wf = init_discover_wf(name="discover_wf",
+                          use_aroma=False,ignore_aroma_err=False,
                           bold_file_size_gb=3)
 
 Given a motion-corrected fMRI, a brain mask, MCFLIRT movement parameters and a
 segmentation, the `discover_wf` sub-workflow calculates potential
 confounds per volume.
+Optional (--use_aroma): give a motion corrected fMRI in MNI space, and mask in MNI space,
+the `discover_wf` sub-workflow calculates potential motion related
+independent components using ICA_AROMA
 
 Calculated confounds include the mean global signal, mean tissue class signal,
 tCompCor, aCompCor, Framewise Displacement, 6 motion parameters and DVARS.
 
+Optional (--use_aroma): aggressive ICA confounds classified by ICA-AROMA.
+Non-aggressive confounds will not be given in the confounds tsv, but denoising can be completed
+by the user. A noise component classification csv and a melodic mix tsv will be passed
+into the derivatives directory. To get non-aggressively denoised data, use the command
+below as a guide:
+
+``fsl_regfilt -i <BIDS SUBJECT>_task-<TASK>_bold_space-<SPACE>_preproc.nii.gz
+-f $(cat <BIDS SUBJECT>_task-<TASK>_bold_AROMAnoiseICs.csv)
```

Comment:
Nope, that's reasonable. Just making sure that the line was doing what I thought it was.